src/MongoDB/Driver/Cursor.cpp

```diff
@@ -55,6 +55,27 @@ const StaticString s_MongoDBDriverCursor_current_doc("current_doc");
 const StaticString s_MongoDBDriverCursor_server_id("server_id");
 IMPLEMENT_GET_CLASS(MongoDBDriverCursorData);
 
+static bool hippo_cursor_load_current(MongoDBDriverCursorData* data);
+
+Object hippo_cursor_init(mongoc_cursor_t *cursor, mongoc_client_t *client)
+{
+	static HPHP::Class* c_result;
+
+	c_result = HPHP::Unit::lookupClass(HPHP::s_MongoDriverCursor_className.get());
+	assert(c_result);
```

Comment:
Yes. If it doesn't work, it should bail out immediately. It's an old remnant, and everywhere else I do this too.